### ISSUE TITLE
AI-123: New github issue created for checking auto Jira ticket creation

### ISSUE THREAD

**Issue by balajiramachandran-seagate** [balajiramachandran-seagate/automate.io#115]
Testing auto Jira ticket creation

**Comment by johnbent** [comment]
Wow @balajiramachandran-seagate , this is awesome!  What happens when I comment here in the github Issue?

**Comment by balajiramachandran-seagate** [comment]
> John Bent commented in Jira:

Wowbala ji, this is awesome!  What happens when I comment here?  Does this only work for atlassaian in the cloud or can it also connect to our on-prem JIRA?

**Comment by balajiramachandran-seagate** [comment]
> John Bent commented in Jira:

Oh wow.  When I comment in github, it appears in JIRA!  That’s awesome!!!!

**Comment by balajiramachandran-seagate** [comment]
> bala ji commented in Jira:

Yes John, we can connect it to our On-Prem using our tokens or password settings. It is not a big deal if we get the approval from IT for using Board Genius Bot.

Also , if we have 30 repos we need to create 30 Bots. It is like for each repo we need a Bot to sync with Github Issue.

The good part is, at low price we can achieve all our use cases.

On Wed, Sep 23, 2020 at 11:16 PM Matt @ Board Geniushelp@boardgenius.iowrote:Hi,Good to chat with you earlier! We can offer you a plan with**100 sync bots for $199/month.**We provide unlimited priority email support for any questions you may have. We offer an onboarding phone call if you need help getting set up.As a reminder, the lower plan I mentioned before is**45 bots for $99/month**.If you have any other questions,please let me know!

Best,MattBoardGenius.io

**Comment by balajiramachandran-seagate** [comment]
> bala ji commented in Jira:

Attached the POC document for reference.

**Comment by balajiramachandran-seagate** [comment]
> bala ji commented in Jira:

Demo given
